bundles/org.openhab.ui/web/src/components/cards/glance/location/status-badge.vue

```diff
@@ -135,20 +135,22 @@ export default {
           allPoints = allEquipmentPoints(equipment)
           points = [
             ...findPoints(allPoints, 'Point_Status', false),
-            ...findPoints(allPoints, 'Point_Control', true)
+            ...findPoints(allPoints, 'Point_Control', true),
+            ...findPoints(allPoints, 'Point_Switch', false)
```

Comment:
Switch is a child of Control I believe, so I don't think you need it explicitly if you check for the existence of Point_Control.

Reply:
As below..